### PR TITLE
Added togglebar at top to see all(Active+Deactive) users, Added Employee page to view and add Employee

### PR DESCRIPTION
**Pull request by opu183059**
### Work Completed List
- Added only Customers in the guest lookup page
- Filtered Deactivated Guest data.
- Account Deactivation or Activation message changed according to button click.
- Active-Deactive Guest Toggle button added to the write above the action button.
- Updated the booking summary function to remove deactivated guest from the suggestions.
- Employee page added and employee page added

### CODE REVIEW

**Comment by opu183059** [comment]
**Is this design ok?** I moved it to the right, above action buttons

ss1

**Comment by opu183059** [comment]
**1. Guest Lookup page showing all active guest by default**

guest_lookup_active_guest

**2. by clicking toggle button showing all (Active+Deactive) users**

guest_lookup_all_guest

**3. Employee page showing only employees**

employee_page

**4. Employee Card Modal**

employee_modal

File: src/pages/GuestLookUp/GuestLookUp.tsx [hunk 1]
```diff
@@ -230,6 +244,18 @@ const GuestLookUp = () => {
             value={searchText}
           />
         </div>
+        <Tooltip
+          title={`See all ${filterDeactivated ? "Active" : "All"} Guest's`}
```
Comment (ahmad-ml) [inline src/pages/GuestLookUp/GuestLookUp.tsx:248]: Guest's >> Guests

File: src/pages/Employees/Employees.tsx [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+      employeeData?.name?.toLowerCase().includes(lowercaseSearchText) ||
+      employeeData?.phone?.toLowerCase().includes(lowercaseSearchText) ||
+      employeeData?.address?.toLowerCase().includes(lowercaseSearchText) ||
+      employeeData?.idType?.toLowerCase().includes(lowercaseSearchText) ||
+      employeeData?.idNo?.toLowerCase().includes(lowercaseSearchText)
+    );
+  });
+  // create employe mutation
+  const [createContact] = useMutation(CREATE_CONTACT, {
+    refetchQueries: [{ query: GET_CONTACTS }],
```
Comment (ahmad-ml) [inline src/pages/Employees/Employees.tsx:90]: get only employees.

File: src/pages/Employees/Employees.tsx [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+  });
+
+  // update Employee function
+  const handleUpdate = async (values: Contact, employeeID: string) => {
+    try {
+      await updateContact({
+        variables: {
+          updateContactInput: {
+            ...values,
+            _id: employeeID,
```
Comment (ahmad-ml) [inline src/pages/Employees/Employees.tsx:104]: There is no employee ID. Plrease remove from the UI

Comment (opu183059) [reply src/pages/Employees/Employees.tsx:104]: Solved the spelling mistake

File: src/pages/Employees/Employees.tsx [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+            allowClear
+            size="middle"
+            onChange={(e) => setSearchText(e.target.value)}
+            value={searchText}
+          />
+        </div>
+
+        <Tooltip
+          title={`See all ${
+            filterDeactivated ? "Acive" : "Deactivated"
```
Comment (ahmad-ml) [inline src/pages/Employees/Employees.tsx:327]: there is no deactivated query. Use Activated and All employees.

File: src/pages/GuestLookUp/GuestLookUp.tsx (hunk 1 above)
Comment (opu183059) [reply src/pages/GuestLookUp/GuestLookUp.tsx]: solved the spelling mistake

File: src/pages/Employees/Employees.tsx (hunk 2 above)
Comment (opu183059) [reply src/pages/Employees/Employees.tsx:90]: Sir this is **Create Employee** function and I have added **Type Employee** when creating content from this employee page, here is the code.

Screenshot 2023-10-20 150107

File: src/pages/Employees/Employees.tsx (hunk 4 above)
Comment (opu183059) [reply src/pages/Employees/Employees.tsx]: Solved, Added Active and all employees

**Comment by ahmad-ml** [comment]
image


Project is not loading ..

**Pull request merged**